README.md

```diff
@@ -86,4 +86,34 @@ client.listData();
 //   },
 //   timestamp: '2019-03-18T14:42:03.192Z',
 // }]
+```
+
+## listDataByDevice(id): &lt;Array&gt;
+
+Get the messages sent by a specific device.
+
+### Result
+- `messages` **Array** JSON object containing device data messages.
+
+### Example
+
+```javascript
+const KNoTCloudStorage = require('@cesarbr/knot-cloud-sdk-js-storage');
+const client = new KNoTCloudStorage({
+  protocol: 'https',
+  hostname: 'data.knot.cloud',
+  id: 'b1a1bd58-c3ef-4cb5-82cd-3a2e0b38dd21',
+  token: '3185a6c9d64915f6b468ee8043df4af5f08e1933',
+});
+
+client.listDataById('cc5429a29afcd158');
```

Comment:
Put a `console.log` so that when the user copies this code, it can see the output shown below.

Reply:
Done.